don't show messages like these on console:
[1:20:0718/174555:ERROR:ffmpeg_demuxer.cc(1577)] OnReadFrameDone result=-541478725 IsMaxMemoryUsageReached=0


diff --git a/media/filters/ffmpeg_demuxer.cc b/media/filters/ffmpeg_demuxer.cc
index 2cf751d..53b0438 100644
--- a/media/filters/ffmpeg_demuxer.cc
+++ b/media/filters/ffmpeg_demuxer.cc
@@ -1591,8 +1591,8 @@ void FFmpegDemuxer::OnReadFrameDone(ScopedAVPacket packet, int result) {
   // - either underlying ffmpeg returned an error
   // - or FFMpegDemuxer reached the maximum allowed memory usage.
   if (result < 0 || IsMaxMemoryUsageReached()) {
-    LOG(ERROR) << __func__ << " result=" << result
-               << " IsMaxMemoryUsageReached=" << IsMaxMemoryUsageReached();
+//    LOG(ERROR) << __func__ << " result=" << result
+//               << " IsMaxMemoryUsageReached=" << IsMaxMemoryUsageReached();
     // Update the duration based on the highest elapsed time across all streams
     // if it was previously unknown.
     if (!duration_known_) {
